Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -107,16 +134,16 @@ public MobiledgeXIntegration(CarrierInfo carrierInfo = null, NetInterface netInt
         /// RegisterClientException and FindCloudletException will give more details on reason for failure
         /// </summary>
         /// <returns>bool Task</returns>
-        public async Task<bool> RegisterAndFindCloudlet(string dmeHost = null, uint dmePort = 0)
+        public async Task<bool> RegisterAndFindCloudlet()
         {
-            bool registered = await Register(dmeHost, dmePort);
+            bool registered = await Register();
```

Comment:
hmm. Didn't we want the dmeHost and dmePort for QA to be able to test?

Reply:
Actually, yes, they're optional parameters at the C# SDK level for testing reasons. If the overrides are expansive enough, or "user specified" override, that can be enough, though I tend to just want optional parameters that are stateless, rather than contextual, which implies state management.